Code/Source/ROS2SystemComponent.cpp

```diff
@@ -86,6 +90,17 @@ namespace ROS2
         m_staticTFBroadcaster = AZStd::make_unique<tf2_ros::StaticTransformBroadcaster>(m_ros2Node);
         m_dynamicTFBroadcaster = AZStd::make_unique<tf2_ros::TransformBroadcaster>(m_ros2Node);
 
+        AZ_Printf("FOOROS2SystemComponent", "Hello!\n");
+        auto* passSystem = AZ::RPI::PassSystemInterface::Get();
```

Comment:
Is this the right place for this? I understand this is something required for the camera sensor.

Reply:
Sorry for debugging - I forgot to remove it. 
Yes, we need to add it here please refer other gems with custom passes: https://github.com/o3de/o3de/blob/development/Gems/Terrain/Code/Source/Components/TerrainSystemComponent.cpp#L70